Python/chapter03/p03_stack_of_plates/miguelHx.py

```diff
@@ -130,54 +130,69 @@ def __str__(self):
         return '->'.join(values)
 
 
+class SetOfStacksIterator(Generic[T]):
+    def __init__(self, stack_list: List[MyStack[T]]):
+        # build temporary list
+        self.my_list = []
+        self.index = 0
+        stack_list_reversed = stack_list[::-1]
+        for stack in stack_list_reversed:
```

Comment:
FYI: this is equivalent:

```suggestion
        for stack in reversed(stack_list):
```

Reply:
Committed suggestion 👍